app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchTEActivity.java

```diff
@@ -205,12 +206,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 v.requestFocus();
             }
             if (event.getAction() == MotionEvent.ACTION_UP) {
+                hideKeyboard();
                 v.clearFocus();
                 v.performClick();
             }
             return true;
         });
 
+        if (binding.navigationBar != null) {
```

Comment:
why there is a possibility that navigationBar is null?.
This activity has SearchTEPresenter, what if we do needed checkos on the presenter and then a method to showNavigationBar?